sdk_extensions/logging/src/main/java/io/opentelemetry/sdk/logging/export/BatchLogProcessor.java

```diff
@@ -70,7 +70,7 @@ public CompletableResultCode forceFlush() {
 
   private static class Worker implements Runnable {
     static {
-      Meter meter = OpenTelemetry.getMeter("io.opentelemetry.sdk.logging");
+      Meter meter = OpenTelemetry.getGlobalMeter("io.opentelemetry.sdk.logging");
```

Comment:
this will probably be something we want to inject, rather than grabbing it globally. follow-on PRs are fine.